packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.ts

```diff
@@ -639,7 +639,8 @@ export const ContractType = (() => {
     const getCancellation = (contract_type: string, cancellation_duration: string, symbol: string) => {
         const arr_cancellation_range: string[] =
             getPropertyValue(available_contract_types, [contract_type, 'config', 'cancellation_range']) || [];
-
+        const cached_arr_cancellation: string[] =
```

Comment:
If the user chose another contract, this `arr_cancellation_range` will be cleaned up and `cancellation_range_list.length === 0` even if it is `Volatility symbol`. That is why we need to cache info about deal cancellation (is it available or not)